test/lib/ansible_test/_internal/units/__init__.py

```diff
@@ -2,12 +2,16 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import json
 import os
 import sys
 
+from packaging.version import Version
```

Comment:
Parse the version without using `packaging`, since it is not part of the Python standard library.

You could probably use this function:

https://github.com/ansible/ansible/blob/13aef3c2e591631440830b6a023e90292a11392b/test/lib/ansible_test/_internal/util.py#L868-L870

However, that will need to be backported as well.

Reply:
I didn't know that function existed. I thought about writing something very similar to it, but wasn't sure what would happen if there was a "weird" version (like `5.4.0b1` or something) which `packaging.version` seems to handle. It looks like pytest only uses standard versioning, so I suppose it will be okay here.

Reply:
If we do need more complex parsing we'll need to implement our own, since `ansible-test` can't normally have direct dependencies on code outside the Python standard library.